apps/prairielearn/python/prairielearn.py

```diff
@@ -759,7 +776,15 @@ def numpy_to_matlab(np_object, ndigits=2, wtype="f"):
         return matrix_str
 
 
-def string_from_numpy(A, language="python", presentation_type="f", digits=2):
+FormatLanguage = Literal["python", "matlab", "mathematica", "r", "sympy"]
```

Comment:
Nit, should we add a leading `_` to indicate that this isn't part of our stable API?